docs/source/creating-mechanisms/configuration_files.rst

```diff
@@ -0,0 +1,40 @@
+#################################
```

Comment:
This is GEOS-Chem specific and not needed.  I have since moved this content to geos-chem.readthedocs.io.